internal/internal_workers_test.go

```diff
@@ -36,6 +32,9 @@ import (
 	"go.uber.org/yarpc"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zaptest"
+	"sync/atomic"
```

Comment:
Check your goimports/gofmt settings since the original version was correct - standard Go packages should go first.